qt/aqt/switch.py

```diff
@@ -1,5 +1,7 @@
 # Copyright: Ankitects Pty Ltd and contributors
 # License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+from __future__ import annotations
```

Comment:
Very minor: I prefer a newline after the comments, but I'm aware we don't do that everywhere already.